pkg/controller/controller.go

```diff
@@ -105,6 +105,8 @@ type ControllerParams struct {
 	NoErrorRetry bool
 
 	Context context.Context
+
+	DelayUpdate time.Duration
```

Comment:
Suggestion: call this jitter? And do please add a docblock describing the purpose.